### PR TITLE
`upload_aqt_configs` methods and downgrade python from 3.8 to 3.7

### PR DESCRIPTION
**Pull request by vtomole**
Fixes: https://github.com/SupertechLabs/cirq-superstaq/issues/15

### CODE REVIEW

**Review by singular-value — CHANGES_REQUESTED** [review]
Just one request

File: cirq_superstaq/service.py [hunk 1]
```diff
@@ -177,3 +177,25 @@ def aqt_compile(self, circuit: cirq.Circuit) -> "cirq_superstaq.aqt.AQTCompilerO
...
+
+        Args:
+            pulses_file_path: The filepath for Pulses.yaml
+            variables_file_path: The filepath for Variables.yaml
+        Returns:
+            A dictionary of of the status of the update (Whether or not it failed)
+        """
+        pulses_file = open(pulses_file_path)
+        read_pulses = pulses_file.read()
+        pulses_file.close()
```
[the remark this answers is not in the document]
Comment (singular-value) [reply cirq_superstaq/service.py:192]: The with block also ensures that the file is closed even if there was an error in the reading
https://realpython.com/read-write-files-python/#:~:text=The%20with%20statement%20automatically%20takes%20care%20of%20closing%20the%20file%20once%20it%20leaves%20the%20with%20block%2C%20even%20in%20cases%20of%20error.%20I%20highly%20recommend%20that%20you%20use%20the%20with%20statement%20as%20much%20as%20possible%2C%20as%20it%20allows%20for%20cleaner%20code%20and%20makes%20handling%20any%20unexpected%20errors%20easier%20for%20you.

[the remark this answers is not in the document]
Comment (vtomole) [reply cirq_superstaq/service.py]: Done.

File: cirq_superstaq/service_test.py [hunk 2]
```diff
@@ -99,6 +99,24 @@ def test_service_aqt_compile(mock_aqt_compile: mock.MagicMock) -> None:
...
+    "cirq_superstaq.superstaq_client._SuperstaQClient.aqt_upload_configs",
+    return_value={"status": "Your AQT configuration has been updated"},
+)
+def test_service_aqt_upload_configs(mock_aqt_compile: mock.MagicMock) -> None:
+    service = cirq_superstaq.Service(remote_host="http://example.com", api_key="key")
+
+    with open("/tmp/Pulses.yaml", "w") as pulses_file:
+        pulses_file.write("Hello")
+
+    with open("/tmp/Variables.yaml", "w") as pulses_file:
```
Comment (singular-value) [inline cirq_superstaq/service_test.py:112]: `variables_file` here and next line

**Review by singular-value — CHANGES_REQUESTED** [review]

File: cirq_superstaq/service_test.py (hunk 2 above)
Comment (vtomole) [reply cirq_superstaq/service_test.py]: whoops copy paste. Done.

**Pull request merged**